x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/NoServicesMessage.test.tsx

```diff
@@ -7,19 +7,20 @@
 import { shallow } from 'enzyme';
 import React from 'react';
 import { NoServicesMessage } from '../NoServicesMessage';
+import { FETCH_STATUS } from '../../../../hooks/useFetcher';
 
 describe('NoServicesMessage', () => {
-  it('should show only a "not found" message when historical data is found', () => {
-    const wrapper = shallow(
-      <NoServicesMessage isLoading={false} historicalDataFound={true} />
-    );
-    expect(wrapper).toMatchSnapshot();
-  });
-
-  it('should show a "no services installed" message, a link to the set up instructions page, a message about upgrading APM server, and a link to the upgrade assistant when NO historical data is found', () => {
-    const wrapper = shallow(
-      <NoServicesMessage isLoading={false} historicalDataFound={false} />
-    );
-    expect(wrapper).toMatchSnapshot();
+  Object.values(FETCH_STATUS).forEach(status => {
```

Comment:
are these useful tests? I guess they don't hurt either, but it feels like it's just generating code paths and then generating snapshots of all the different code paths. The component itself seems to be simple (and will likely stay that way) to warrant snapshot tests.

Reply:
I felt it was an okay-ish way to test the different code paths since the generated snapshots were rather small (I agree that changes in big snapshots are often overlooked). What do you suggest instead?